INSTALL

```diff
@@ -296,12 +296,22 @@ sudo tar -xf core-image-weston-MACHINE_NAME-xt.tar.bz2 -C /srv/domu/
 It worth mentioning that some changes to domain config files are necessary to switch between NFS
 and any of storage devices.
 
-For doing that:
-cd workspace_base/meta-xt-prod-gen3-test/recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/
+There are two options for doing that. Either by modifying source files followed by rebuilding
+the whole Domain-0's image or by modifying already installed files which implies unpacking an image
```

Comment:
...the whole Domain-0' or by modifying domain config file of DomD right in a built initramfs image of Dom0, which....